Desktop/widgets/boundcontrolterms.cpp

```diff
@@ -130,14 +130,14 @@ void BoundControlTerms::bindTo(Option *option)
 Option* BoundControlTerms::createOption()
 {
 	Option* result = nullptr;
-	if (_listView->hasRowComponent() || _termsModel->areTermsInteractions())
+	if (_listView->hasRowComponent() || _listView->containsInteractions())
 	{
 		Options* templote = new Options();
 
-		if (_termsModel->areTermsInteractions())
+		if (_listView->containsInteractions())
 		{
 			OptionTerm * newOptTerm = new OptionTerm();
-			newOptTerm->setShouldEncode(true);
+			newOptTerm->setShouldEncode(_listView->containsVariables());
```

Comment:
I assume this is that superclass then (It isnt easy to see here in the review part of github sorry)

Reply:
yep